tldw_Server_API/tests/AuthNZ/integration/test_single_user_claims_permissions.py

```
@@ -0,0 +1,266 @@
+    assert ok_second is True
+
+    settings = get_settings()
+    single_user_key = settings.SINGLE_USER_API_KEY
+
+    async def _limited_principal(request: Request):
+        principal = AuthPrincipal(
+            kind="single_user",
+            user_id=settings.SINGLE_USER_FIXED_ID,
+            api_key_id=None,
+            subject=None,
+            token_type="api_key",
```

Ruff (0.14.6)

Possible hardcoded password assigned to argument: "token_type"

(S106)